### PR TITLE
N37: Cybergate

### PR DESCRIPTION
**Pull request by cybersoulK**
Part of #1134

### CODE REVIEW

**Comment by cybersoulK** [comment]
i am not sure if the last 3 bullet points are necessary. 
Feel free to remove and change what may be necessary

File: content/news/037/index.md [hunk 1]
```diff
@@ -91,6 +91,33 @@ refactoring tool based on the [Language Server Protocol][language-server-protoco
...
+
+CyberGate ([YouTube][cybergate-yt], [Discord][cybergate-dis]) by CyberSoul
+is a new multiplayer project that aims at procedurally generating distinct
+universes and gameplay experiences. CyberGate is the name of the main world
+where universes can be created and accessed by quantum portals.
+
+Recent updates:
+
+- Ported to browser (wgpu with webgl backend)
+- Unreliable protocol archieved with WebRTC
```
Comment (17cupsofcoffee) [inline content/news/037/index.md:107]: Is this meant to say 'achieved'?

**Review by 17cupsofcoffee — APPROVED** [review]
lgtm, just one bit i'm not sure about

File: content/news/037/index.md (hunk 1 above)
Comment (cybersoulK) [reply content/news/037/index.md:107]: yep achieved. my bad

**Review by ozkriff — CHANGES_REQUESTED** [review]
lgtm expect for the image size

> Only include one image (<300kb) or GIF (<2.5mb). 

Could you optimize it a little, please?

**Review by ozkriff — APPROVED** [review]
:rocket:

**Pull request merged**